Sources/NIO/SelectableEventLoop.swift

```diff
@@ -59,6 +59,12 @@ internal final class SelectableEventLoop: EventLoop {
     @usableFromInline
     internal var _scheduledTasks = PriorityQueue<ScheduledTask>()
     private var tasksCopy = ContiguousArray<() -> Void>()
+    @usableFromInline
+    internal var _succeededVoidFuture: Optional<EventLoopFuture<Void>> = nil {
+        didSet {
+            assert(self.inEventLoop)
```

Comment:
As an incredibly minor detail: can we use the `assertInEventLoop` function? I _know_ that on this specific event loop it doesn't matter, but it's nice to eliminate this particular anti-pattern from the codebase where we can.

Reply:
oh yes!

Reply:
fixed, thank you! Will merge when green